### PR TITLE
Vendor protobuf and go extension issue PR

### PR DESCRIPTION
**Pull request by rajatlko13**
@ferranbt please checkout this

### CODE REVIEW

File: scripts/tools-protobuf.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.12.0/protoc-3.12.0-linux-x86_64.zip
+unzip protoc-3.12.0-linux-x86_64.zip -d $HOME/.local
+export PATH="$PATH:$HOME/.local/bin"
+
+go get google.golang.org/grpc
```
Comment (ferranbt) [inline scripts/tools-protobuf.sh:5]: I do not think we need to vendor grpc at all, that is already done in go modules.

File: scripts/tools-protobuf.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+sudo apt-get install -y golang-protobuf-extensions-dev
```
Comment (ferranbt) [inline scripts/tools-protobuf.sh:7]: What is this for?

**Review by ferranbt — CHANGES_REQUESTED** [review]

File: scripts/tools-protobuf.sh (hunk 2 above)
Comment (rajatlko13) [reply scripts/tools-protobuf.sh:7]: You asked me to add golang protobuf extension, I thought this is the one we needed.

Comment (rajatlko13) [reply scripts/tools-protobuf.sh:7]: github.com/protocolbuffers/protobuf-go

Comment (ferranbt) [reply scripts/tools-protobuf.sh:7]: Not that one either, please try this:
```

File: scripts/tools-protobuf.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.12.0/protoc-3.12.0-linux-x86_64.zip
+unzip protoc-3.12.0-linux-x86_64.zip -d $HOME/.local
+export PATH="$PATH:$HOME/.local/bin"
```
Comment (ferranbt) [inline scripts/tools-protobuf.sh:3]: Do not add extra things on PATH, add it to /usr/local/bin or something like that

File: scripts/tools-protobuf.sh (hunk 2 above)
Comment (ferranbt) [reply scripts/tools-protobuf.sh:7]: Also please try to run ```make protoc``` after this to see if it works or not.

**Comment by ferranbt** [comment]
So far looks good but there is something wrong if the protobuf files were regenerated completely. Please run the next commands to see which versions have been installed.
```
$ protoc-gen-go-grpc --version
$ protoc-gen-go --version
$ protoc --version
```

**Review by victorcoder — DISMISSED** [review]
LGTM

**Review by ferranbt — APPROVED** [review]

**Pull request merged**